src/components/ha-generic-picker.ts

```diff
@@ -149,14 +127,18 @@ export class HaGenericPicker extends LitElement {
   private _unsubscribeTinyKeys?: () => void;
 
   protected render() {
-    return html`
-      ${this.label
-        ? html`<label ?disabled=${this.disabled}>${this.label}</label>`
-        : nothing}
-      <div class="container">
+    const hasValue = !!this.value?.length;
```

Comment:
```suggestion
```

why the `length`? When a string is empty it's false.
I would get rid of this variable. It's just used once and there it's negated.